gui/wxpython/core/toolboxes.py

```diff
@@ -789,9 +789,10 @@ def new_translator(string):
     sys.displayhook = new_displayhook
     sys.__displayhook__ = new_displayhook
 
-    import __builtin__
+    if sys.version_info[0] == 2:
+        import __builtin__
 
-    __builtin__._ = new_translator
+        __builtin__._ = new_translator
```

Comment:
The "underscore" issue should still apply, so I would be surprised if doing it just for 2 is enough. Besides that, if there is 2-only code somewhere, it should be deleted. (But I think that's not the case here.)

Reply:
Right. We do not support Python 2 anymore, so it is probably even better to remove Python 2 specific code to cleanup (also other places where Python 2 code pops up)?